atapas/nodeX
# feat: nodex project browser interface

how to test:

just checkout to this branch and run:

`node build.js`

a new (ignored) folder will appear with the following structure:

```
index.html
app.js
styles.css
```

what's new:

- created a build.js script to generate a final html with each module examples and topics nested in an accordion structure
- less than 10ms to run and being available into an ignored build folder
- no angular/react/vue/jekill or any external libraries/frameworks included, because seems to be a little over engineering for this project (just used built-in libraries)
